test/e2e/kafka_binding_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/google/uuid"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/eventing-contrib/test/e2e/helpers"
-	"knative.dev/eventing/test/lib"
+	testlib "knative.dev/eventing/test/lib"
```

Comment:
I saw some duplicates of this import. Not duplicated by your commits, but in case you want to fix them.
Or, I can send a separate PR.

Reply:
@aliok I'm sorry can you tell me more what is a duplicate? I'm afraid I didn't get it